docs/docs/APIGuide/Triggers.md

```diff
@@ -95,3 +95,26 @@ s reached.
  A trigger that triggers an action when training loss
  less than "min" loss
 
+---
+## And
+**Scala:**
+```scala
+  val trigger = Trigger.and(Trigger.minLoss(0.01), Trigger.maxEpoch(2))
+```
+**Python:**
+```python
+  trigger = And(MinLoss(0.01), MaxEpoch(2))
+``` 
+A trigger contains other triggers and triggers when all of them trigger (logical AND)
+
+---
+## Or
+**Scala:**
+```scala
+  val trigger = Trigger.and(Trigger.minLoss(0.01), Trigger.maxEpoch(2))
+```
+**Python:**
+```python
+  trigger = Or(MinLoss(0.01), MaxEpoch(2))
+``` 
+A trigger contains other triggers and triggers when all of them trigger (logical OR)
```

Comment:
Comments not clear. Actually, user can use this api to do early stop.

Reply:
As early stop is just a specific use case of Or, so I add an example to explain this usage.